tests/core/variables/test_annualize.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+from pytest import fixture
+
+from openfisca_core import periods
+from openfisca_core.model_api import *  # noqa analysis:ignore
+from openfisca_country_template.entities import Person
+from openfisca_core.variables import get_annualized_variable
+
+
+@fixture
+def monthly_variable():
+    class monthly_variable(Variable):
+        value_type = int
+        entity = Person
+        definition_period = MONTH
+        nof_call = 0
```

Comment:
Is this an abbreviation for "number of calls" ? If so, I'd suggest a different name, this leaves too much room for interpretation.

Reply:
I used to think the same, until I recently learned from native English speakers that `nb` is [**not** an abbreviation for number in English](https://forum.wordreference.com/threads/abbreviation-of-number-n-n%C2%B0-nr-nbr-no.264328/), but only for _nota bene_.
"No" is the standard abbreviation, but `no_calculations` would be awfully confusing.

I'm ok with switching to `calculation_count` though